.github/workflows/publish-js-sdks.yml

```diff
@@ -17,20 +17,26 @@ jobs:
           node-version-file: "package.json"
           registry-url: "https://registry.npmjs.org"
 
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          run_install: true
+
       - name: Install dependencies
-        run: npm install
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js install
```

Comment:
Again this is redundant with line 23 above (I generally recommend against a scoped install, it might seem like it would be an optimization but it actually usually just makes it harder to do caching well in CI and doesn't usually buy you very much even in the best case)